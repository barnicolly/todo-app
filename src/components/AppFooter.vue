<template>
  <div>
    <div>
      <small>
        Элементы можно переносить, редактирование по клику на текст элемента
      </small>
    </div>
    <div>
      <small>
        Исходный код:&nbsp;
        <a target="_blank" href="https://github.com/barnicolly/vue-todo-app">
          https://github.com/barnicolly/vue-todo-app
        </a>
      </small>
    </div>
  </div>
</template>
<script setup></script>
