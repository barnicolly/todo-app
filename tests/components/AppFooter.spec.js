import { shallowMount } from '@vue/test-utils';
import AppFooter from '@/components/AppFooter.vue';

describe('AppFooter.vue Snapshot', () => {
  it('has same HTML structure', () => {
    const wrapper = shallowMount(AppFooter);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
