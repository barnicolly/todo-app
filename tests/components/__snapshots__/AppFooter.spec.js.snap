// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppFooter.vue Snapshot has same HTML structure 1`] = `
<div>
  <div><small> Элементы можно переносить, редактирование по клику на текст элемента </small></div>
  <div><small> Исходный код:&nbsp; <a target="_blank" href="https://github.com/barnicolly/vue-todo-app"> https://github.com/barnicolly/vue-todo-app </a></small></div>
</div>
`;
